### PR TITLE
add update notice to an outdated blog post

### PR DESCRIPTION
**Pull request by alisaduncan**
Update an outdated blog post redirecting to dev docs

### CODE REVIEW

**Comment by mraible** [comment]
The preview for this is at https://deploy-preview-1357--okta-blog.netlify.app/blog/2018/06/08/add-authentication-to-any-web-page-in-10-minutes. If you look at the text, it's not accurate since it says the link points to an updated version of the blog post. 

Screenshot 2023-04-04 at 9 23 32 AM

**Comment by alisaduncan** [comment]
> The preview for this is at https://deploy-preview-1357--okta-blog.netlify.app/blog/2018/06/08/add-authentication-to-any-web-page-in-10-minutes. If you look at the text, it's not accurate since it says the link points to an updated version of the blog post.
> 
> Screenshot 2023-04-04 at 9 23 32 AM

Yes, I could not find a blog post that I thought was a match. All of the blog posts were framework specific (and the one about adding schematics is Auth0 based). So I went with docs.

I can look into creating a different embed for redirecting to docs. If it take too much time/effort to implement, I'll will need to go with this as we are trying to address an issue where this post is getting a lot of attention right now.

Meant to tag @mraible

**Comment by alisaduncan** [comment]
@mraible I updated if you wouldn't mind taking checking it out

File: _source/_posts/2018-06-08-add-authentication-to-any-web-page-in-10-minutes.md [hunk 1]
```diff
@@ -15,11 +15,26 @@ tweets:
...
 
+<div class="alert alert-info" role="alert">
+  <p class="title">
+    <a class="close" aria-label="Close" title="Close"
+       onclick="this.parentElement.parentElement.style.display='none'">
+      <span aria-hidden="true">&times;</span>
+    </a>
+    Heads up... this blog post is old!
+  </p>
+  <p>For an updated instructions on adding authentication using the Okta Sign-in Widget, see
```
Comment (mraible) [inline _source/_posts/2018-06-08-add-authentication-to-any-web-page-in-10-minutes.md]:
```suggestion
  <p>For updated instructions on adding authentication using the Okta Sign-in Widget, see
```

**Comment by alisaduncan** [comment]
@mraible updated

**Review by mraible — APPROVED** [review]

**Pull request merged**